packages/builder/builder-rspack-provider/tests/plugins/minimize.test.ts

```diff
@@ -107,16 +172,48 @@ describe('plugins/minimize', () => {
       origin: { bundlerConfigs },
     } = await builder.inspectConfig();
 
-    expect(bundlerConfigs[0].builtins?.minifyOptions).toMatchInlineSnapshot(`
-      {
-        "asciiOnly": true,
-        "extractComments": true,
-        "pureFuncs": [
-          "console.log",
-          "console.warn",
-        ],
-      }
-    `);
+    expect(bundlerConfigs[0].optimization?.minimizer).toMatchInlineSnapshot(
+      `
+      [
+        Plugin {
```

Comment:
How can I make this be `SwcJsMinimizerPlugin`, is this read `[Symbol.toString]`?

Reply:
it seems get from constructor's name, maybe you can define a name for SwcJsMinimizerPlugin class?

Reply:
Disregard me, it's not constructor's name...

Reply:
Actually, I don't know, it seems to take the class name

![20230907-170619](https://github.com/web-infra-dev/modern.js/assets/22373761/ed827614-7e3b-48a7-98a8-fdba627ca6d0)

but, if you add a instance name,this also can be output like this:

<img width="426" alt="image" src="https://github.com/web-infra-dev/modern.js/assets/22373761/1f30dfa6-39b3-45fe-a66e-a705b1892f55">